cmd/ui.go

```diff
@@ -380,3 +385,13 @@ func yamlPrint(w io.Writer, v interface{}) error {
 	}
 	return nil
 }
+
+// checkWebDashboardDescription returns true if desc contains web dashboard description.
+// The returned string contains info string (URL).
+func checkWebDashboardDescription(desc string) (bool, string) {
+	const webDashboardDescPrefix = webDashboardName + " "
+	if strings.HasPrefix(desc, webDashboardDescPrefix) {
+		return true, strings.TrimPrefix(desc, webDashboardDescPrefix)
+	}
+	return false, ""
+}
```

Comment:
As the extension is going to be included in the core, can we just not include the name in it so we can simplify the code here? IIRC, the description is not used somewhere else.

Reply:
I followed the `desc == engine.IngesterDescription` example here. I agree that the use of description is unfortunate, the name would be better. By the way, the name is not hard-coded into the xk6-dashboard, it is named "web-dashboard" in the k6 source code. Its original name is "dashboard" in the xk6-dashboard source code. Thus, the name included in the extension is not included in the core source code, but the name defined in the core source code is used in several places in the core source code.
Let me know if it would be good if I check the name here instead of the description.

Reply:
I simplified and eliminated the check in a separate function

Reply:
>By the way, the name is not hard-coded into the xk6-dashboard, it is named "web-dashboard" in the k6 source code. Its original name is "dashboard" in the xk6-dashboard source code.

It sounds like we should align them. Otherwise, we will have this difference, and in the long term it could create issues. I think could be a good idea to define an exported variable directly in the dashboard package as `const dashboard.OutputName = "web dashboard"`. For example, here https://github.com/grafana/xk6-dashboard/blob/6a4cbea58a4fd2a5984c66f6893411e3736c1eae/register.go#L18.

In this way, we are sure both use the same name consistently.

Reply:
I would prefer this constant to remain in the k6 repo. The xk6-dashboard output extension defines output as "dashboard". For the xk6-dashboard built into k6, k6 defines an output as "internal-web-dashboard" (I renamed it), and we defined this name here. That is, during the integration into k6, we define a name for the output that we only use here. It is logical that this constant should be in the k6 repo.

Reply:
>I would prefer this constant to remain in the k6 repo.

Fine for me.

>The xk6-dashboard output extension defines output as "dashboard". For the xk6-dashboard built into k6, k6 defines an output as "internal-web-dashboard"

I still think this is an error and will generate confusion. @oleiade @mstoykov it would be good to get your opinion here.

Reply:
done, xk6-dashboard now use same name, "web-dashboard" and exports a constant for it